databus-client2/src/main/java/com/bazaarvoice/emodb/databus/client2/Json/JsonUtil.java

```diff
@@ -0,0 +1,74 @@
+package com.bazaarvoice.emodb.databus.client2.Json;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.google.common.base.Throwables;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class JsonUtil {
+
+    private final static ObjectMapper OBJECT_MAPPER = new ObjectMapper()
+            .registerModule(new Jdk8Module())
+            .registerModule(new JavaTimeModule());
+
+    public static <T> T parseJson(String json, Class<T> type) {
+        try {
+            return OBJECT_MAPPER.readValue(json, type);
+        } catch (IOException e) {
+            throw Throwables.propagate(e);
```

Comment:
nitpick, to not depend on guava here and below could be replaced with 
```suggestion
            throw new RuntimeException(e);
```